packages/webdriver/tests/command.test.ts

```diff
@@ -75,6 +75,31 @@ const scope: BaseClient = new FakeClient()
 type mockResponse = (...args: any[]) => any
 
 describe('command wrapper', () => {
+    beforeEach(() => {
+        delete process.env.KEEP_WDIO_LOGGER
+    })
+    
+    function doProperRequest (additionalParams: number | undefined = undefined) {
+        const commandFn = commandWrapper(commandMethod, commandPath, commandEndpoint)
+        const requestMock = require('../src/request/node')
+        const resultFunction = commandFn.call(scope, '123', 'css selector', '#body', additionalParams) as unknown as mockResponse
+        expect(requestMock.mock.calls).toHaveLength(1)
+        expect(resultFunction({ value: 14 })).toBe(14)
+        if (additionalParams) {
+            expect(resultFunction({ value: 'foobarboo' })).toBe('foobarboo')
+        }
+
+        const [method, endpoint, { using, value, customParam }] = requestMock.mock.calls[0]
+        expect(method).toBe('POST')
+        expect(endpoint).toBe('/session/:sessionId/element/123/element')
+        expect(using).toBe('css selector')
+        expect(value).toBe('#body')
+        if (additionalParams) {
+            expect(customParam).toBe(additionalParams)
+        }
+        requestMock.mockClear()
+    }
+    
```

Comment:
I don't think these tests are relevant to the change you made above, let's revert them.

Reply:
reverted